### ISSUE TITLE
Field Report for cake-autorate 3.0.1 on D-Link DIR-878

### ISSUE THREAD

**Issue by richb-hanover** [lynxthecat/cake-autorate#238]
Using D-Link DIR-878 with OpenWrt 22.03.5; generating traffic from Rasperry Pi 4b connected via Ethernet. My ISP connection is 75/75 mbps. I used the idle value from `top` for most measurements because `htop` doesn't summarize the CPU usage across multiple cores.

* Installation was unremarkable. Installer reported version 3.0.1.

* I configured to use ifb4wan and wan ports (DL/UL), set my speeds as required

* I started manually (`./cake-autorate.sh`).

a) Run out of box;  lots of logging; `htop` shows CPU runs 15-25% when idle; passing traffic goes to 80-85% (I was probably only observing one of the cores, though. Nonetheless, lots of spare CPU.)

b)  I turned off logging (output_processing_stats, output_load_stats, output_reflector_stats, debug set to zero; output_cake_changes set to 1) `htop` showed  CPU 12-22% (slightly lower) when idle; 60-75% with traffic. (Again, probably only recording one core, but plenty of spare CPU)

c) Turn off cake-autorate (leaving SQM running); `top` shows 85% idle no traffic; ~50% idle with traffic 

d) Turn off SQM; `top` shows 99% idle when no traffic; 79% idle with traffic  

**TL;DR** The multi-core DIR-878 has plenty of CPU to handle a 75/75 mbps ISP connection with cake-autorate installed. I have left it running as a service.

**Comment by rany2** [comment]
That's great news for a 800Mhz mt7621 CPU. Thanks!

**Comment by lynxthecat** [comment]
Agreed. @moeller0 write about the MT7621A here: 

https://forum.openwrt.org/t/cake-w-adaptive-bandwidth/135379/2977?u=lynx

Actually my outdoor 5G router - Zyxel NR7101 has something called the MT7621AT. Is that the same processor?

**Comment by richb-hanover** [comment]
> With htop configure it to show all CPUs and then look at the idle percentsge of the CPU with the highest load (or simply pick the lowest idle number reported), once the cake cpu isfully loaded you reached the limit independent of the load on the other CPUs....

Cool tip. My `htop` configuration simply shows (what I believe is) utilization on each of the cores. 

* Is it useful to report the highest of the four?
* Or is there a better configuration for `htop`? 

Thanks.

<img width="649" alt="image" src="https://github.com/lynxthecat/cake-autorate/assets/1094930/38a8bfdb-61fd-441c-85b7-a818db080240">

**Comment by moeller0** [comment]
Not strictly better, but I typically enable the detailed CPU graph that shows software interrupt processing separately, I also add an additoonal textual display for each CPU. However currently I am travelling without a real computer and can not test how to configure things 'my way' nor create/post a screenshot I think...

**Comment by moeller0** [comment]
Not strictly better, but I typically enable the detailed CPU graph that shows software interrupt processing separately (F2, Display Options , Detailed CPU time), I also add an additonal textual display for each CPU (F2, Meters, -ECANNOTTEST). However currently I am travelling without a real computer and can not test how to configure things 'my way' nor
